examples/common-chunk-grandchildren/README.md

```diff
@@ -0,0 +1,469 @@
+```
+
+</details>
+
+``` javascript
+/******/ ([
+/* 0 */
+/*!********************!*\
+  !*** ./example.js ***!
+  \********************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports, __webpack_require__) {
+
+var main = function() {
+	console.log("Main class");
+	__webpack_require__.e/* require.ensure */(2).then((() => {
+		const page = __webpack_require__(/*! ./pageA */ 3);
+		page();
+	}).bind(null, __webpack_require__)).catch(__webpack_require__.oe);
+	__webpack_require__.e/* require.ensure */(1).then((() => {
+		const page = __webpack_require__(/*! ./pageB */ 4);
+		page();
+	}).bind(null, __webpack_require__)).catch(__webpack_require__.oe);
+};
+
+main();
+
+
+/***/ }),
+/* 1 */
+/*!***************************************!*\
+  !*** multi ./example.js ./example.js ***!
+  \***************************************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports, __webpack_require__) {
+
+__webpack_require__(/*! ./example.js */0);
+module.exports = __webpack_require__(/*! .\example.js */0);
+
+
+/***/ }),
+/* 2 */
+/*!******************************!*\
+  !*** ./reusableComponent.js ***!
+  \******************************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports) {
+
+module.exports = function() {
+	console.log("reusable Component");
+};
+
+
+/***/ })
+/******/ ]);
+```
+
+# js/0.output.js
+
+``` javascript
+webpackJsonp([0],{
+
+/***/ 5:
+/*!******************!*\
+  !*** ./pageC.js ***!
+  \******************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports, __webpack_require__) {
+
+var reusableComponent = __webpack_require__(/*! ./reusableComponent */ 2);
+
+module.exports = function() {
+	console.log("Page C");
+	reusableComponent();
+};
+
+
+/***/ })
+
+});
+```
+
+# js/1.output.js
+
+``` javascript
+webpackJsonp([1],{
+
+/***/ 4:
+/*!******************!*\
+  !*** ./pageB.js ***!
+  \******************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports, __webpack_require__) {
+
+module.exports = function() {
+	console.log("Page B");
+	__webpack_require__.e/* require.ensure */(0).then((()=>{
+		const page = __webpack_require__(/*! ./pageC */ 5);
+		page();
+	}).bind(null, __webpack_require__)).catch(__webpack_require__.oe);
+};
+
+
+/***/ })
+
+});
+```
+
+# js/2.output.js
+
+``` javascript
+webpackJsonp([2],{
+
+/***/ 3:
+/*!******************!*\
+  !*** ./pageA.js ***!
+  \******************/
+/*! no static exports found */
+/*! all exports used */
+/***/ (function(module, exports, __webpack_require__) {
+
+var reusableComponent = __webpack_require__(/*! ./reusableComponent */ 2);
+
+module.exports = function() {
+	console.log("Page A");
+	reusableComponent();
+};
+
+
+/***/ })
+
+});
+```
+
+# Info
+
+## Uncompressed
+
+```
+Hash: [1mfd4796594a8274a0759b[39m[22m
```

Comment:
This should not include the color special chars. Try to pass `--no-colors` in examples/build-common.js to fix it.